### PR TITLE
New lens helper for React users

### PR DESCRIPTION
**Pull request by stellarhoof**
Given that React is so popular, export a utility function to convert a
`useState` call to a lens

### CODE REVIEW

**Comment by decrapifier** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Browser test error. Please see CI logs for more details.</td>
    </tr>
  </tbody>
</table>



<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>We were able to automatically fix some formatting issues in this PR for you!</td>
    </tr>
  </tbody>
</table>


### Some things that were possibly fixed:

- Code that could be fixed via the --fix flag
- Formatting that could be fixed by prettier

Take a look at this commit to see what happened in detail: d7e450664e3c626a4ecd39e6a72b749ae8bd43b0


And look at this wiki page to see the reasoning behind the ESLint rules: https://github.com/smartprocure/eslint-config-smartprocure/wiki/Rules-and-Why-We-Chose-Them
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against da012b88def0a5e8719e1913a9159f41a0a02f00
</p>

**Comment by decrapifier** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Browser test error. Please see CI logs for more details.</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against b78123aa2b0c689609be4cafbfa3312ccdc2b6c7
</p>

File: karma.conf.js [hunk 1]
```diff
@@ -269,14 +269,6 @@ let customLaunchers = {
     deviceName: 'iPad Pro (9.7 inch) Simulator',
     deviceOrientation: 'portrait',
   },
-  sl_android_44: {
```
Comment (stellarhoof) [inline karma.conf.js]: https://wiki.saucelabs.com/display/DOCS/2019/03/07/Announcing+End+of+Life+for+old+OS+versions+on+Sauce+emulators+and+simulators

**Comment by michsa** [comment]
This was my favorite option and I'm glad it won 👍

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
-﻿# 1.55.1
```
Comment (daedalus28) [inline CHANGELOG.md]: This should be a minor release because it’s a new feature

**Pull request merged**